syntax = "proto3";

package API.Union;

import "common.proto";

message GetInfo {
    int32 server_id = 1;
    int32 char_id = 2;
}

message GetInfoDone {
    int32 ret = 1;
    repeated API.Common.ExtraReturn extras = 2;

    string union_id = 3;
    int32 owner_id = 4;
}
